addon/components/a11y-landmark.js

```diff
@@ -60,5 +87,22 @@ export default Ember.Component.extend({
         }
     }),
 
+    _validateProperties() {
+        this._validateTagName(this.tagName);
+        this._validateLandmarkRole(this.landmarkRole);
+    },
 
-});
\ No newline at end of file
+    _validateTagName(tagName) {
```

Comment:
Yup! We can certainly do DRY things up if they stay in this form. I wanted to get feedback first and refactor things like this afterwards. It's more work if we DRY things up, then have to pull them back apart.